WTWSMS/common/landed_titles/WtWSMS_zusk_titles.txt

```diff
@@ -302,14 +305,7 @@ e_khitan = {
 	}
 	
 	dynasty_title_names = no 	# Will not be named "Seljuk", etc.
-	
-	# Regnal names	
-	male_names = {
-		Aidi Aizong Bala Chengzong Chongzong Daizong Daozong Dezong Dingzong Duanzong Duzong Gaozong Gaozu Gongzong Guangzong Guizong 
-		Hendeji Houzhu Huaizong Huanzong Huizong Hula Jingzong Kangzong Liezu Lizong Mingzong Muzong Ningzong Qinzong Renzong Ruizong 
-		Shangdi Shaozhu Shenzong Shizong Shizu Shunzong Su Suzong Taizong Tianzuo Wa Wenzong Wuwang Wuzong Xiangzong Xianzhi Xianzhu Xianzong Xiaozong 
-		Xingzong Xizong Xuanzong Yelan Yingzong Yizong Yuanzong Zhangzong Zhaogu Zhaozong Zhengzong Zhenzong Zhongzhu Zhongzong Zhuangzong Zuwu
-	}
```

Comment:
Show me some history saying that a Khitan Emperor changed his name upon assuming the throne, and I'll add them back.